<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StdSlope</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Mesh"/>
    <addaction name="actionImport_Mesh_East_North_Up"/>
    <addaction name="actionImport_Mesh_West_Down_South"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuPanels">
     <property name="title">
      <string>Panels</string>
     </property>
    </widget>
    <addaction name="menuPanels"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionReload"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Mesh">
   <property name="text">
    <string>Import Mesh</string>
   </property>
  </action>
  <action name="actionset">
   <property name="text">
    <string>set</string>
   </property>
  </action>
  <action name="actionLayers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Layers</string>
   </property>
  </action>
  <action name="actionStereonet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stereonet</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionImport_Mesh_East_North_Up">
   <property name="text">
    <string>Import Mesh (East, North, Up)</string>
   </property>
  </action>
  <action name="actionImport_Mesh_West_Down_South">
   <property name="text">
    <string>Import Mesh (West, Down, South)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
